### PR TITLE
Feature/ATG-5 Add EsLint

### PR DESCRIPTION
**Pull request by IraCherginyshka**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "build": "ng build",
     "test": "ng test",
     "lint": "ng lint",
+    "eslint": "eslint --ext .js,.ts ./src --fix" ,
```
Comment (oleksandrstarov) [inline package.json]: Please, remove extra space before comma

File: package.json [hunk 2]
```diff
@@ -7,6 +7,7 @@
...
     "e2e": "ng e2e"
```
Comment (oleksandrstarov) [inline package.json]: Please remove all e2e settings (now it is not possible to run ng lint command for example as ti tries to find e2e files as well)

File: src/app/app.component.spec.ts [hunk 3]
```diff
@@ -20,7 +20,7 @@ describe('AppComponent', () => {
     expect(app).toBeTruthy();
   });
 
-  it(`should have as title 'proj'`, () => {
+  it('should have as title \'proj\'', () => {
```
Comment (oleksandrstarov) [inline src/app/app.component.spec.ts]: Proj was a temporary project name - please update this test

**Review by oleksandrstarov — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (IraCherginyshka) [reply package.json]: Done

File: package.json (hunk 2 above)
Comment (IraCherginyshka) [reply package.json]: Done

File: src/app/app.component.spec.ts (hunk 3 above)
Comment (IraCherginyshka) [reply src/app/app.component.spec.ts]: Done

**Review by oleksandrstarov — APPROVED** [review]

**Pull request merged**